### PR TITLE
Autologin Feature Added

### PR DESCRIPTION
**Pull request by koushikmln**
Login by passing username and password in url as query parameters. 
Migrated to node-pty which is currently maintained and updated. 
added feature to put in global username and password
Added server keep alive in ssh options to keep the session alive.

### CODE REVIEW

**Comment by butlerx** [comment]
Could you rebase also quite a few users run this in docker so that would be a requirement

**Comment by koushikmln** [comment]
What is the exact issue you are facing on docker? Can you give me some logs if possible? I will also try setting up and try it out.

**Comment by koushikmln** [comment]
@butlerx added docker compatibility and rebased code.

File: public/wetty/index.html [hunk 1]
```diff
@@ -3,7 +3,9 @@
   <head>
     <meta charset="UTF-8">
     <title>Wetty - The WebTTY Terminal Emulator</title>
+    <script src="/wetty/hterm_all.js"></script>
```
Comment (butlerx) [inline public/wetty/index.html]: this shouldn't be required it should be bundled in `wetty.js`

File: public/wetty/index.html [hunk 2]
```diff
@@ -3,7 +3,9 @@
...
     <script src="/wetty/socket.io/socket.io.js"></script>
+    <script src="/wetty/wetty.js"></script>
```
Comment (butlerx) [inline public/wetty/index.html]: this is required at the bottom of the file

**Review by butlerx — CHANGES_REQUESTED** [review]

**Comment by koushikmln** [comment]
My bad. Fixed it.

**Comment by butlerx** [comment]
thanks for that

**Comment by koushikmln** [comment]
Let me know if you have some features in mind. We can discuss on email and collaborate.

**Pull request merged**